Added CI instructions.

dave-bell opened the pull request.

Commit:
Added CI instructions.

dave-bell commented:
Not sure about the image tag.  Smartcall at least is fetching `1.0` during its build, I think down stream projects should just be pulling `master`.

mrbaboon commented on Makefile:
```diff
@@ -0,0 +1,55 @@
+GIT_COMMIT ?= local-commit
+BRANCH_NAME ?= local-branch
+
+VERSION=$(shell docker-compose run --rm smartcall version)
```
I think this will fail.

In this project's case, we can stick it in deploy.env and it will get spat into the environment here.

mrbaboon commented on deploy.env:
```diff
@@ -0,0 +1,3 @@
+IMAGE_NAME=healthcheck-test
+REGISTRY_URI=595212869906.dkr.ecr.us-east-1.amazonaws.com
+AWS_CLI_REGION=us-east-1
```
Put version in here for now.

dave-bell replied:
Versioning was removed outright, no need to update this file with that content.

mrbaboon commented on Jenkinsfile:
```diff
@@ -0,0 +1,58 @@
+    ])
+])
+
+def projectName = 'smartcall'
```
change project name

dave-bell replied:
duh, thanks for catching that.

Commit:
PR changes:
1) corrected project name
2) removed version related tags and pushes as it isn't relevant to this project.
